UnitTests/OIDTokenRequestTests.m

```diff
@@ -239,6 +239,10 @@ - (void)testSecureCoding {
   XCTAssertNotNil(request.additionalHeaders, @"");
   XCTAssertEqualObjects(request.additionalHeaders[kTestAdditionalHeaderKey],
                         kTestAdditionalHeaderValue, @"");
+  
+  NSURLRequest *urlRequest = [request URLRequest];
+  XCTAssertEqualObjects([urlRequest.allHTTPHeaderFields objectForKey:kTestAdditionalHeaderKey],
```

Comment:
You can remove the third parameter here (`@""`) since it does nothing. I know that is the existing style in this file, but let's start moving away from this practice starting now!

Reply:
Thank you. Is this issue resolved or is there something I need to do on my end?